build/config/genesis.json

```diff
@@ -1,146 +1,149 @@
 {
-  "consensus": {
-    "genesis_time": {
-      "seconds": 1660514050,
-      "nanos": 296019000
-    },
-    "chain_id": "testnet",
-    "max_block_bytes": 4000000
-  },
-  "utility": {
-    "pools": [
+  "PersistenceGenesisState": {
```

Comment:
Yes, because protos need to be unique.

See TODO for research:

`// TODO (Research) is there anyway to not have to name these protobuf files uniquely? `